README.md

```diff
@@ -2,62 +2,51 @@
 
 # End to end tests for Archive.org using [Playwright](https://playwright.dev/)
 
+
 ## (Optional) BrowserStack Local Setup
 
-- login to BrowserStack and retrieve the account username and access key, see documentation [here](https://www.browserstack.com/docs/automate/playwright/getting-started/nodejs/test-runner)
+- Login to BrowserStack and retrieve the account username and access key, see documentation [here](https://www.browserstack.com/docs/automate/playwright/getting-started/nodejs/test-runner)
 
 
-## Running tests locally
+## Local Setup
 
-- install dependencies:
+- Install dependencies:
 
     `npm i`
 
-- install Playwright browser libs:
+- Install Playwright browser libs:
 
     `npx playwright install`
 
-- run all the tests in headless mode and generate 1 whole test report:
-
-    `npm run test`
+- Create a `.env` file by copying the contents from `.env.sample` and add the respective values you want to use for testing
 
-- run all the tests in headless mode by each category and generate test report by category:
+    - this is required to run tests with loggedIn flows
 
-    `./run-tests.sh`
 
-- run all the tests in headed mode (this will load multiple browsers):
+## Running tests
 
-    `npm run test:headed`
+- Run all tests: `npm run test`
 
-- create a `.env` file by copying the contents from `.env.sample` and add the respective values you want to use for testing
+- Run test by category (sample):
 
+    - This will run a custom script that can take different shell arguments/parameters for the following (arguments arrangement can be jumbled or not in proper order):
```

Comment:
Questions: 
- Was this "jumbled parameters" feature asked for?
- How do we run a single test by grep of title?
- How do we run a subset or tagged tests (login/not login, fast/slow, dark/normal identifiers)
- Can we pass in identifiers?

Please take your time and think through the possibilities. The method doesn't have to do everything at first, but it should be easily changeable, extendable.